pkg/kubelet/types/types_test.go

```diff
@@ -23,6 +23,99 @@ import (
 	"k8s.io/kubernetes/pkg/api/v1"
 )
 
+func TestConvertToTimestamp(t *testing.T) {
+	var cases = []struct {
```

Comment:
It seems that the case table is not needed here. At least, we can use string slice.